.env.vault.dist

```diff
@@ -0,0 +1,3 @@
+POSTGRES_CONTAINER=mypostgres
+POSTGRES_USER=vault
+POSTGRES_PASSWORD=vault
```

Comment:
can you change keys ?
something like: `POSTGRES_VAULT_USER` and `POSTGRES_VAULT_PASSWORD` seems better to don't have any conflict.